### PR TITLE
chore(tracing): expose tracing api

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: docs/src/api/class-tracing.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+```python async
+await context.tracing.start(name="trace" screenshots=True snapshots=True)
+await page.goto("https://playwright.dev")
+await context.tracing.stop()
+await context.tracing.export("trace.zip")
+```
+
+```python sync
+context.tracing.start(name="trace" screenshots=True snapshots=True)
```
Comment (dgozman) [inline docs/src/api/class-tracing.md:47]: Same here

File: docs/src/api/class-tracing.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+page.goto("https://playwright.dev")
+context.tracing.stop()
+context.tracing.export("trace.zip")
+```
+
+### option: Tracing.start.name
+- `name` <[string]>
+
+If specified, the trace is going to be saved into the file with the
```
Comment (dgozman) [inline docs/src/api/class-tracing.md:56]: In what directory will the file be saved? I think we are missing docs for `traceDir` option?

File: src/cli/cli.ts [hunk 3]
```diff
@@ -159,21 +159,25 @@ commandWithOpenOptions('pdf <url> <filename>', 'save page as pdf',
...
+      if (command.browser === 'ff')
+        command.browser = 'firefox';
+      if (command.browser === 'wk')
+        command.browser = 'webkit';
+      showTraceViewer(trace, command.browser).catch(logErrorAndExit);
+    }).on('--help', function() {
+      console.log('');
+      console.log('Examples:');
+      console.log('');
+      console.log('  $ show-trace --resources=resources trace/file.trace');
```
Comment (dgozman) [inline src/cli/cli.ts:178]: Drop `--resources`

File: src/cli/cli.ts [hunk 4]
```diff
@@ -159,21 +159,25 @@ commandWithOpenOptions('pdf <url> <filename>', 'save page as pdf',
...
-      }).on('--help', function() {
-        console.log('');
-        console.log('Examples:');
-        console.log('');
-        console.log('  $ show-trace --resources=resources trace/file.trace');
-        console.log('  $ show-trace trace/directory');
-      });
-}
+program
+    .command('show-trace [trace]')
```
Comment (dgozman) [inline src/cli/cli.ts:163]: Is `[trace]` a file or a directory?

Comment (pavelfeldman) [reply src/cli/cli.ts:163]: I think it'll be both.

**Pull request merged**